airflow/models/baseoperator.py

```diff
@@ -955,7 +955,7 @@ def __deepcopy__(self, memo):
 
         for k, v in self.__dict__.items():
             if k not in shallow_copy:
-                setattr(result, k, copy.deepcopy(v, memo))  # noqa
+                setattr(result, k, v if type(v).__name__ == 'module' else copy.deepcopy(v, memo))   # modules cannot be pickled
```

Comment:
What module is being copied? We've already got the shallow_copy_attrs list -- so anything not deep-copyable should probably go in that list

Reply:
Good point. I am not sure if this comes from the different implementations of some dependencies between Windows and the other systems. This was basically one approach to get rid of the errors popping up without having to go through all the offending attributes. I will see if I can make this a bit less "hacky".